x-pack/plugins/security_solution/cypress/e2e/entity_analytics/entity_analytics_management_page.cy.ts

```diff
@@ -0,0 +1,119 @@
+import { getNewRule } from '../../objects/rule';
+import { createRule } from '../../tasks/api_calls/rules';
+import { waitForAlertsToPopulate } from '../../tasks/create_new_rule';
+import {
+  SHOW_DATES_BUTTON,
+  DATE_PICKER_ABSOLUTE_TAB,
+  DATE_PICKER_ABSOLUTE_INPUT,
+  LOCAL_DATE_PICKER_START_DATE_POPOVER_BUTTON,
+  LOCAL_DATE_PICKER_END_DATE_POPOVER_BUTTON,
+  LOCAL_DATE_PICKER_APPLY_BUTTON_TIMELINE,
+} from '../../screens/date_picker';
+
+describe(
+  'Entity analytics management page',
+  { env: { ftrConfig: { enableExperimental: ['riskScoringRoutesEnabled'] } } },
+  () => {
+    before(() => {
+      esArchiverResetKibana();
+      cleanKibana();
+    });
+
+    beforeEach(() => {
+      login();
+      esArchiverLoad('all_users');
+      visitWithoutDateRange(ALERTS_URL);
+      createRule(getNewRule({ query: 'user.name:* or host.name:*', risk_score: 70 }));
+      waitForAlertsToPopulate();
+      visit(ENTITY_ANALYTICS_MANAGEMENT_URL);
+    });
+
+    after(() => {
+      esArchiverUnload('all_users');
+    });
+
+    it('renders page as expected', () => {
+      cy.get(`${PAGE_TITLE}`).should('have.text', 'Entity Risk Score');
+    });
+
+    describe('Risk preview', () => {
+      it('there is data for risk and host preview react on date range', () => {
+        const START_DATE = 'Jan 18, 2019 @ 20:33:29.186';
+        const END_DATE = 'Jan 19, 2019 @ 20:33:29.186';
+
+        cy.get(HOST_RISK_PREVIEW_TABLE_ROWS).should('have.length', 5);
+        cy.get(USER_RISK_PREVIEW_TABLE_ROWS).should('have.length', 5);
+
+        cy.get('.euiSuperDatePicker');
+        cy.get('body').then(($container) => {
+          if ($container.find(SHOW_DATES_BUTTON).length > 0) {
+            cy.get(SHOW_DATES_BUTTON).click({ force: true });
+          } else {
+            cy.get(LOCAL_DATE_PICKER_START_DATE_POPOVER_BUTTON).click({ force: true });
+          }
+        });
+
+        cy.get(DATE_PICKER_ABSOLUTE_TAB).first().click({ force: true });
+
+        cy.get(DATE_PICKER_ABSOLUTE_INPUT).click();
+        cy.get(DATE_PICKER_ABSOLUTE_INPUT).clear();
+        cy.get(DATE_PICKER_ABSOLUTE_INPUT).type(START_DATE);
+
+        cy.get(LOCAL_DATE_PICKER_APPLY_BUTTON_TIMELINE).click();
+        cy.get(LOCAL_DATE_PICKER_APPLY_BUTTON_TIMELINE).should('not.have.text', 'Updating');
+
+        cy.get(LOCAL_DATE_PICKER_END_DATE_POPOVER_BUTTON).click({ force: true });
+
+        cy.get(DATE_PICKER_ABSOLUTE_TAB).first().click({ force: true });
+
+        cy.get(DATE_PICKER_ABSOLUTE_INPUT).click();
+        cy.get(DATE_PICKER_ABSOLUTE_INPUT).clear();
+        cy.get(DATE_PICKER_ABSOLUTE_INPUT).type(END_DATE);
+        cy.get(LOCAL_DATE_PICKER_APPLY_BUTTON_TIMELINE).click();
+
+        cy.get(HOST_RISK_PREVIEW_TABLE).contains('No items found');
+        cy.get(USER_RISK_PREVIEW_TABLE).contains('No items found');
+      });
+
+      it('Show error panel if API returns error', () => {
```

Comment:
I've the feeling this test is doing more that what it says due to the action we are performing after checking the `Preview failed` text. I would split the test or write a better name for it.